### PR TITLE
LPS-154215 Remove LIFERAY_LIB_GLOBAL_SHARED_DIR since it is no longer needed

### PR DESCRIPTION
**Pull request by julschong**
Hi Tina, 
This is resubmitted from: #808 

Note: the change is to inline calculating for globalLibDir inside `StartupAction` from this comment: https://github.com/liferay-core-infra/liferay-portal/pull/808#discussion_r888239265

### CODE REVIEW

File: portal-impl/src/com/liferay/portal/events/StartupAction.java [hunk 1]
```diff
@@ -70,8 +74,20 @@ protected void doRun(String[] ids) throws Exception {
 		// Check Tomcat's lib/ext directory
 
 		if (ServerDetector.isTomcat()) {
-			File libExtDir = new File(
-				PropsValues.LIFERAY_LIB_GLOBAL_SHARED_DIR, "ext");
+			String path = ClassUtil.getParentPath(
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/events/StartupAction.java:77]: @julschong Please try not to use ClassUtil, let's try to simplify the logic here as much as possible.

Thanks.
Tina.

**Comment by julschong** [comment]
forced pushed after avoiding using ClassUtil.getParentDir() to keep StartupAction simpler

File: portal-impl/src/com/liferay/portal/events/StartupAction.java [hunk 2]
```diff
@@ -70,8 +72,13 @@ protected void doRun(String[] ids) throws Exception {
 		// Check Tomcat's lib/ext directory
 
 		if (ServerDetector.isTomcat()) {
-			File libExtDir = new File(
-				PropsValues.LIFERAY_LIB_GLOBAL_SHARED_DIR, "ext");
+			String appServerDir = StringUtil.replace(
+				System.getProperty("catalina.base"), CharPool.BACK_SLASH,
+				CharPool.FORWARD_SLASH);
+
+			String globalSharedDir = appServerDir + "/lib/";
```
Comment (julschong) [inline portal-impl/src/com/liferay/portal/events/StartupAction.java:79]: Changed here

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
c

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:acceptance

**Comment by julschong** [comment]
rebased and resolved conflict for bnd.bnd and packageinfo

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
forced pushed to resolve conflict for: 

> portal-impl/bnd.bnd
> portal-kernel/bnd.bnd

**Comment by julschong** [comment]
force pushed to resolve conflcit for:

> portal-impl/bnd.bnd

**Comment by tinatian** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/933

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**